serverless/deploy_cpu.sh

```diff
@@ -4,19 +4,23 @@
 SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
 FUNCTIONS_DIR=${1:-$SCRIPT_DIR}
 
-nuctl create project cvat
+docker build -t cvat.openvino.base "$SCRIPT_DIR/openvino/base"
+
+nuctl create project cvat --platform local
 
 shopt -s globstar
 
 for func_config in "$FUNCTIONS_DIR"/**/function.yaml
 do
-    func_root=$(dirname "$func_config")
-    echo Deploying $(dirname "$func_root") function...
-    nuctl deploy --project-name cvat --path "$func_root" \
-        --volume "$SCRIPT_DIR/common:/opt/nuclio/common" \
-        --platform local
-done
+    func_root="$(dirname "$func_config")"
+    func_rel_path="$(realpath --relative-to="$SCRIPT_DIR" "$(dirname "$func_root")")"
```

Comment:
Should not we updated deploy_gpu.sh the same way?

Reply:
Right, thanks for the reminder. I ported relevant changes to `deploy_gpu.sh`.